### ISSUE TITLE
Question about experiment results

### ISSUE THREAD

**Issue by taeyeopl** [Junggy/HouseCat6D#7]
Thanks for sharing a great work. I have some questions related to the results. 

Q1. How do you train and report the NOCS, FS-net, and GPV-Pose results since Vi-Net implementations are only provided? I'd like to know the training strategy, whether train per class per model or train a single model to estimate all the classes.

Q2. I'd like to know the difference in training between NOCS and HouseCat6D and the results on the REAL275 or House6D Dataset. Have you tried or used the pre-trained model (e.g., NOCS, FS-net, GPV-Pose) if the classes (e.g., Bottle on the NOCS dataset) are the same during the evaluation?

Q3. Is the same class(e.g., bottle, can, cup) of pose compatible with the NOCS protocol, or does it have a different reference pose definition?

**Comment by Junggy** [comment]
Thanks for your interest on our dataset, and sorry for the late reply!

A1. we adapted NOCS, FS-net and GPV pose's dataloader for our own dataset and trained from the scratch. As far as I see, one category level network is one single model that predicts pose on all categories. During training for example, GPV pose and FSNet reads image and randomly pick one object instance from any category into a batch that one input batch contains combination of instances from random category.

A2. I guess our A1 answers the question. We train them from the scratch using housecat dataset.

A3. For the nocs protocol, check our supplementary material Fig.10-13. We design the canonical orientation of each category to be standing with y axis (or y is a symmetric axis). But its hard to say they are compatible (or not sure what does compatible means here), as some categories have different shapes. I assume, categories like cup, teapot, bottle, glass, tube can would be kinda compatible as they are share the cylindrical shape as base. But not for box/remote/shoe.

**Comment by taeyeopl** [comment]
Thanks for your replies. 

Do you plan to release the training and evaluation code for grasp pose estimation on your dataset? Is it possible to reproduce the results presented in Table 5 (a), including the pre-trained grasp pose model?

**Comment by Junggy** [comment]
@ymxlzgy can help you out with any questions regarding grasping

**Comment by taeyeopl** [comment]
@Junggy @ymxlzgy Do you have any updates?

**Comment by Junnn-create** [comment]
> Thanks for your replies.
> 
> Do you plan to release the training and evaluation code for grasp pose estimation (KGN) on your dataset? Is it possible to reproduce the results presented in Table 5, including the pre-trained grasp pose model?

I have same question. @ymxlzgy Can you please update me on the progress?
